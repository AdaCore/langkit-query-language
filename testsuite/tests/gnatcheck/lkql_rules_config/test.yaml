driver: gnatcheck
project: prj.gpr
lkql_rule_file: config.lkql
format: brief
show_rule: True
control:
  - [SKIP, "mode == 'ada'", "LKQL rule file feature unavailable on Ada implementation"]