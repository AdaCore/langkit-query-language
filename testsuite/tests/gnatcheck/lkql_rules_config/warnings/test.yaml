driver: gnatcheck
format: brief
input_sources:
  - main.adb
tests:
  - label: Classic argument object
    lkql_rule_file: config.lkql
  - label: Shortcut argument
    lkql_rule_file: short.lkql
