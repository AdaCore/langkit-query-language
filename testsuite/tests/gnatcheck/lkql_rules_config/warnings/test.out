Classic argument object
=======================

main.adb:1:09: unit "Ada.Text_IO" is not referenced

Shortcut argument
=================

main.adb:1:09: unit "Ada.Text_IO" is not referenced
